### PR TITLE
Added play/pause and track skip to buttons 2-4.

### PR DESCRIPTION
**Pull request by JeffreyO**
I've 'fixed' the bug that crashes the program when an ad plays due to a NULL reference.

I also cleaned up your Visual Studio Project files and altered some things to make it more convenient and easier to build later on.

Enjoy!

### CODE REVIEW

**Comment by LPardue** [comment]
Thanks!

I'll take a closer look and hope to merge soon.

**Comment by JeffreyO** [comment]
Fixed the bug is a bit of a stretch. I've added a try-catch block temporarily so that the program doesn't crash (And so I get debug output when the exception is thrown.). It still throws the "System.NullReferenceException" exception during the ad, though it still runs rather than crashing. When the program picks up correct track data after the ad, it resumes normal operation as if nothing had happened.

I'm looking for a way to completely eliminate the exception from happening by checking to see if any of the needed values are NULL and substituting in a place-holder value. It should be commited once I have the issue taken care of.

File: SpotifyStatusApplet/SpotifyStatusApplet.csproj [hunk 1]
```diff
@@ -1,167 +1,187 @@
...
+      <Name>SpotifyLocalAPI</Name>
+    </ProjectReference>
+  </ItemGroup>
+  <Import Project="$(MSBuildToolsPath)\Microsoft.CSharp.targets" />
+  <PropertyGroup>
+    <PreBuildEvent>
+    </PreBuildEvent>
+  </PropertyGroup>
+  <PropertyGroup>
+    <PostBuildEvent>xcopy /d /y "$(SolutionDir)\Binaries\*.dll" $(TargertDir)
```
Comment (JeffreyO) [inline SpotifyStatusApplet/SpotifyStatusApplet.csproj:177]: TODO: Fix typo. $(TargertDir) needs to be $(TargetDir). Whoops.

File: README.md [hunk 2]
```diff
@@ -1,40 +1,42 @@
...
-
-Dependencies
---------------------------------------
-
-SpotifyStatusApplet builds upon the great efforts of other developers/projects! The following software components are incorporated into the source tree to aid distribution
-
-- [Spotify Local API](https://code.google.com/p/spotify-local-api/) - A very simple and small library that allows .NET developers to get track information, (un)pause spotify, play tracks, get cover art and more! 
-- [GammaJul LgLcd](http://gjlglcd.codeplex.com/) - A .NET wrapper around the Logitech SDK for G15/G19 keyboard screens. Supports raw byte sending, GDI+ drawing and rendering WPF elements onto the screen.
-- [Json.net](http://json.codeplex.com/) - Json.NET is a popular high-performance JSON framework for .NET.
+SpotifyStatusApplet
```
Comment (JeffreyO) [inline README.md:1]: I believe the diff operation is failing due to changed file permissions. The GitHub application (for Windows 7) had issues synchronizing changes as well, so I had to upload the files to GitHub through the web interface.

**Pull request closed without merging**